subgraphs/uniswap-forks/yarn.lock

```diff
@@ -0,0 +1,5561 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I know we have mixed use of npm and yarn in the repo, but ideally we should stick to just one of them. Since uniswap-forks was only using npm, can your remove the `yarn.lock` from the PR?

Reply:
Makes sense, this was accidental, will remove.